compiler/plc_driver/src/tests/snapshots/plc_driver__tests__external_files__calling_external_file_function_without_including_file_results_in_error.snap

```diff
@@ -2,4 +2,4 @@
 source: compiler/plc_driver/./src/tests/external_files.rs
 expression: msg.to_string()
 ---
-E071: cannot generate call statement for ReferenceExpr { kind: Member(Identifier { name: "external" }), base: None } at: external_file.st:2:8:{2:8-2:16}: 
+cannot generate call statement for ReferenceExpr { kind: Member(Identifier { name: "external" }), base: None } at: external_file.st:2:8:{2:8-2:16}: 
```

Comment:
did we lose the error-no by accident? Don't I need them to ask for explanation?

Reply:
It did not exactly get lost, the problem is this test does not use the diagnostician who would have reported the correct errors. It instead relies on the errors being returned as a Result.
I'll add an issue to better handle diagnostics in tests.